.github/workflows/unit-tests.yaml

```diff
@@ -12,6 +12,39 @@ on:
     - reopened
     - synchronize
 jobs:
+  go-postgres:
+    strategy:
+      matrix:
+        gotags: [ 'GOTAGS=""', 'GOTAGS=release' ]
+    runs-on: ubuntu-latest
+    container:
+      image: quay.io/stackrox-io/apollo-ci:stackrox-test-0.3.49
+    services:
+      postgres:
+        image: ghcr.io/enterprisedb/postgresql:15
```

Comment:
Is there any chance to find quay.io or registry.access.redhat.com alternative?

Reply:
Quick search revealed these:
- https://quay.io/repository/fedora/postgresql-13?tab=info
- https://quay.io/repository/openshift-community-operators/postgresql?tab=info - would require installing the operator on OpenShift in order to figure out where the actual postgres container comes from.

Reply:
The most reliable would be to create RH image pull credentials and use this https://catalog.redhat.com/software/containers/rhel8/postgresql-13/5ffdbdef73a65398111b8362?container-tabs=gti

Reply:
> passing unit tests may not catch problems with real image

So we should use `central-db` image. I'd try to make it work as it has some expectations about environment (e.g. mounted configuration file)

Reply:
I give up, let's copy approach from osci and maybe revisit usage of services later.